site/docs/advanced/deployment.md

```diff
@@ -36,7 +36,7 @@ This depends on your deployment type.
 ### Webhooks
 
 1. Make sure you do not perform any long-running operations in your middleware, such as large file transfers.
-   [This leads to timeout errors](../guide/deployment-types.md#ending-webhook-requests-in-time) for the webhooks, and duplicate update processing as Telegram will re-send non-acknowledged updates.
+   This leads to [timeout errors](../guide/deployment-types.md#ending-webhook-requests-in-time) for the webhooks, and duplicate update processing as Telegram will re-send non-acknowledged updates.
```

Comment:
I'd revert this one. The link is about how one thing leads to another, which is what the section explains. It's not about timeout errors only.